### PR TITLE
Add CORS headers

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
Adds CORS headers required for integration with OTP-RR.

### CODE REVIEW

**Review by philip-cline — APPROVED** [review]

File: handler.ts [hunk 1]
```diff
@@ -91,6 +91,8 @@ export const makePeliasRequests = async (
   return {
     body: JSON.stringify(mergedResponse),
     headers: {
+      'Access-Control-Allow-Credentials': 'true',
+      'Access-Control-Allow-Origin': '*',
```
Comment (binh-dam-ibigroup) [inline handler.ts:95]: Do we need also `Access-Control-Allow-Methods`?

**Review by binh-dam-ibigroup — CHANGES_REQUESTED** [review]
I think `Access-Control-Allow-Methods` is needed too.

File: handler.ts (hunk 1 above)
Comment (miles-grant-ibigroup) [reply handler.ts:95]: I've taken this from the [Serverless CORS guide](https://www.serverless.com/blog/cors-api-gateway-survival-guide/) which seems to suggest that `Access-Control-Allow-Methods` is handled by AWS API Gateway

**Comment by miles-grant-ibigroup** [comment]
Thanks for the review, please see my comment

File: handler.ts (hunk 1 above)
Comment (binh-dam-ibigroup) [reply handler.ts:95]: Ok, please add a comment in that section of the code (and maybe mention the serverless cors setting) for future reference.

**Review by binh-dam-ibigroup — APPROVED** [review]
Approved pending addressing per https://github.com/ibi-group/pelias-stitch/pull/3#discussion_r721426582.

**Pull request merged**